src/cases/view-models/case-detail.view-model.js

```diff
@@ -1,5 +1,93 @@
+import jsonpath from "jsonpath";
 import { getFormattedGBDate } from "../../common/helpers/date-helpers.js";
 
+const findCaseDetailsTab = (overrideTabs) => {
+  return (overrideTabs || []).find((tab) => tab.id === "caseDetails");
+};
+
+const resolvePayloadReference = (ref, payload) => {
+  if (!ref || !payload) {
+    return undefined;
+  }
+
+  try {
+    return jsonpath.value(payload, ref.replace("$.payload.", "$."));
+  } catch (error) {
+    return undefined;
+  }
+};
```

Comment:
Why do we need to do this? If we don't take the payload off the case item on line 60:

`processSectionFields(section, caseItem.payload)`

Just do `processSectionFields(section, caseItem)` then we don't need the `resolvePayloadReference` function?

Also means if we need data from outside the payload we can still access it? (I think there was talk of also using reference data at some point)